src/room.py

```diff
@@ -11,4 +11,4 @@ def __init__(self, name, description):
         self.w_to = None
 
     def __str__(self):
-        return (f"{self.name}\n{self.description}")
\ No newline at end of file
+        return (f"\u001b[35m{self.name}\n\u001b[36m{self.description}\u001b[37m \n")
```

Comment:
Make a commit to showcase MVP first, then work on stretches. This is not an ideal practice.